### PR TITLE
Remove session on logout, route changes, other cleanups

### PR DESCRIPTION
**Pull request by fidian**
On logout the session record is now removed from storage.

The router is updated to allow _account-id to be :accountId.  This was
updated so we can now have /account/_account-id/something/_something-id
in order to keep the IDs separate.

Other things were found and updated as well.

### CODE REVIEW

File: lib/account-manager.js [hunk 1]
```diff
@@ -135,6 +135,18 @@ module.exports = (challengeManager, config, random, sessionManager, storageServi
 
 
     /**
+     * Destroy the session record.
+     *
+     * @param {string} accountId Account ID
```
Comment (AbsentSemicolon) [inline lib/account-manager.js]: Do we need a comment saying Account ID when the parameter is `accountId`? I feel this is self-explanatory. Same with Session ID below this.

File: lib/session-manager.js [hunk 2]
```diff
@@ -25,6 +25,21 @@ module.exports = (config, promise, random, storageServiceFactory) => {
 
 
     /**
+     * Destroys a session
```
Comment (AbsentSemicolon) [inline lib/session-manager.js]: Missing period.

File: lib/account-manager.js (hunk 1 above)
Comment (fidian) [inline lib/account-manager.js]: I completely agree.  Commit 7274227

File: lib/session-manager.js (hunk 2 above)
Comment (fidian) [inline lib/session-manager.js]: Arrrrggggg.  9a499d0

**Comment by AbsentSemicolon** [comment]
Reviewed and looks good.

**Pull request merged**